src/components/icon/styles.css

```diff
@@ -1,3 +1,18 @@
+a.fill-div {
+     display: block;
+	width: 96px;
+	height: 96px;
```

Comment:
indention

Reply:
@Michadelic why resolved when it's not? We use tabs, not spaces, through whole project, this has to be fixed.